MoneyHoney/redux/actionCreater.jsx

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
This file should be `actionCreator` not creater

Reply:
renamed